src/plugins/widgets/workHours/types.ts

```diff
@@ -0,0 +1,13 @@
+import { API } from "../../types";
+
+export type Data = {
+  startTime: string;
+  endTime: string;
+}
```

Comment:
Should we include `workDays: number[]` to track which days they work? You would then hide the widget on non-working days.